tox.ini

```
@@ -1,6 +1,6 @@
 [tox]
 minversion = 2.4.0
-envlist = docs,flake8,mypy,twine,py{36,37,38,39,310},du{14,15,16,17,18}
+envlist = docs,flake8,mypy,twine,coverage,py{36,37,38,39,310},du{17,18,19}
```

```suggestion
envlist = docs,flake8,mypy,twine,coverage,py{37,38,39,310,311},du{17,18,19}
```

And below on line 18.

---

Adding 311 envs should be done on the 5.x branch, I've dropped the outdated envs on this PR.

A

---

Thanks, please see https://github.com/sphinx-doc/sphinx/pull/10871 to add `311` to 5.x.